6_stemming_lematization/Stemming_and_Lemmatization_Solutions.ipynb

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
Cell # 34, Exercise 1. - Write a short note on the words that have different base words using stemming and Lemmatization
In the solution, can you write short note on the words? Basically we have given that as an exercise but there is no solution